server/service/s3_object_presigner/s3_object_presigner.go

```diff
@@ -45,9 +45,9 @@ func (p *S3ObjectPresigner) CheckAndPresign(ctx context.Context) {
 		}
 
 		if s3ObjectPayload.LastPresignedTime != nil {
-			// Skip if the presigned URL is still valid for the next 6 days.
-			// The default expiration time is 7 days.
-			if time.Now().Before(s3ObjectPayload.LastPresignedTime.AsTime().Add(6 * 24 * time.Hour)) {
+			// Skip if the presigned URL is still valid for the next 4 days.
+			// The expiration time is set to 5 days.
+			if time.Now().Before(s3ObjectPayload.LastPresignedTime.AsTime().Add(4 * 24 * time.Hour)) {
```

Comment:
Any context about this change?

Reply:
I saw that the commit https://github.com/usememos/memos/commit/cf7718f8dc98290a36c1210491130ad4b7b0ff63 changed the expiration time but did not update it in this file